package.json

```diff
@@ -122,7 +124,7 @@
     "xregexp": "^2.0.0"
   },
   "devDependencies": {
-    "babel-eslint": "^10.1.0",
+    "@babel/eslint-parser": "^7.17.0",
```

Comment:
fixes `npm WARN deprecated babel-eslint@10.1.0: babel-eslint is now @babel/eslint-parser. This package will no longer receive updates.` warning.

see https://www.npmjs.com/package/@babel/eslint-parser